src/renderer/ui/utils/constants.js

```diff
@@ -48,6 +48,7 @@ export const languageMap = {
   'en-US': 'English',
   'fr-FR': 'French',
   it: 'Italian',
+  ja: '日本語',
```

Comment:
Historically we have the English name of the Language here, so this should be `Japanese`.

Though in the future I guess these strings should be translatable as well 🌵 

But for now (and for consistency) can we set this to be Japanese 😄